go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/filecoin-project/go-f3
 
-go 1.21
+go 1.20
```

Comment:
why?

Reply:
It got bumped by go-broadcast. The CI runs 1.20, lotus runs 1.20.

Reply:
Ah, I see.